### PR TITLE
enable deploy to slot option for linux apps

### PR DESCRIPTION
**Pull request by dikhakha**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ export async function deployWebAppImage(endPoint, resourceGroupName, webAppName)
 
     if(imageSourceAndTag)
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts]: This is not related to your PR. but is this condition required ?

Comment (dikhakha) [inline Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts]: this check is not required here. can be removed.

**Comment by sachinma** [comment]
UTs?

**Comment by vincentdass** [comment]
#4460

File: Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts (hunk 1 above)
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts]: Please do remove the condition, prior to merge.

Comment (dikhakha) [inline Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts]: done
